locality-checker/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"github.com/cockroachdb/k8s/locality-checker/pkg/kubernetes"
+)
+
+const localityMountPath = "/etc/cockroach-locality"
+
+func main() {
+	ctx := context.Background()
```

Comment:
It probably doesn't matter for a container that will have such a small lifespan but it may be worth while to respect TIGTERM by cancelling this context?

Credit to @vladdy  for this one:
```go
func newMainCtx() context.Context {
	ctx, cancel := context.WithCancel(context.Background())
	go func() {
		sCh := make(chan os.Signal, 1)
		signal.Notify(sCh, syscall.SIGINT, syscall.SIGTERM)
		<-sCh
		cancel()
	}()
	return ctx
}
```